src/MadLib.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import MadLib from './MadLib';
+
+it('will render without crashing', () => {
+  render(<MadLib />)
+})
+
+it("matches snapshot", () => {
+  const { asFragment } = render(<MadLib />);
+  expect(asFragment()).toMatchSnapshot();
+});
```

Comment:
I do think a happy path end-to-end test is appropriate here. A reviewer should be able to understand expected behavior at each level of abstraction (i.e., `MadLib`, `MadLibForm`, etc.). So the top-level should be the shortest path from start to finish. Something like:
```suggestion
});

it("let's you play madlibs", () => {
  const { getByText, getAllByRole, asFragment } = render(<MadLib />);
  const firstRender = asFragment()
  
  // Select an option
  fireEvent.change(getByText("story-select"), { target: { value: "One Sentence Wonder" } })
  const afterSelect = asFragment()
  
  // Fill in form
  getAllByRole("input").forEach((input) => {
    fireEvent.change(input, { target: { value: "blah" } })
  }
  const afterFormFill = asFragment()
  
  // Click "show story" button
  // Expect "blah" to be in story element
  // Click "start over" button
  // Expect <select> to be present
});
```